src/Sulu/Bundle/SecurityBundle/DependencyInjection/Configuration.php

```diff
@@ -44,6 +45,40 @@ public function getConfigTreeBuilder()
                 ->end()
             ->end();
 
+
+        $this->addObjectsSection($rootNode);
+
         return $treeBuilder;
     }
+
+    /**
+     * Adds `objects` section.
+     *
+     * @param ArrayNodeDefinition $node
+     */
+    private function addObjectsSection(ArrayNodeDefinition $node)
```

Comment:
@chirimoya i think this is mostly copied to all bundles? shouldnt we put this in a trait like `PersistenceConfigurationTrait `?